app/views/org_admin/phases/container.html.erb

```diff
@@ -63,7 +63,7 @@
                         <% unless phase.sections.all?(&:modifiable?) %>
                           <%= _("You may place them before or after the main template sections.") %>
                         <% end %>
-                      <% else %>
+                      <% elsif template.latest? %>
```

Comment:
So this hides the link to re-order sections on Historic Templates?
Good catch, Just checked on DMPonline and hitting that throws a 404